new-lamassu-admin/src/pages/Triggers/helper.js

```diff
@@ -390,20 +448,34 @@ const type = currency => ({
   }
 })
 
-const requirementSchema = Yup.object().shape({
-  requirement: Yup.object({
-    requirement: Yup.string().required(),
-    suspensionDays: Yup.number().when('requirement', {
-      is: value => value === 'suspend',
-      then: Yup.number()
-        .required()
-        .min(1),
-      otherwise: Yup.number()
-        .nullable()
-        .transform(() => null)
+const requirementSchema = Yup.object()
+  .shape({
+    requirement: Yup.object({
+      requirement: Yup.string().required(),
+      suspensionDays: Yup.number().when('requirement', {
+        is: value => value === 'suspend',
+        then: Yup.number()
+          .required()
+          .min(1),
```

Comment:
I think this validation is not necessary since we're doing it in the test function, right?
Could be replaced by:

 suspensionDays: Yup.number()
   .nullable()
   .transform(transformNumber)